src/Illuminate/Support/Collection.php

```diff
@@ -458,6 +458,17 @@ public function map(callable $callback)
     }
 
     /**
+     * Map a collection and flatten the result by a single level.
+     *
+     * @param  callable $callback
```

Comment:
I'll be writing a fixer for this for StyleCI in a few weeks, but until then, just make sure you have 2 spaces either side of the type for param annotations.